webui/src/components/KeyResult.vue

```diff
@@ -4,6 +4,7 @@
                     autosave="true"
                     acceptEmpty="true"
                     :readonly="readonly"
+                    :multiline="true"
```

Comment:
As far as I understood it is a static property, so `multiline="true"` is enough (no colon required).

Reply:
I didn't know the difference. Will fix